### PR TITLE
How to use the NGINX Docker image

### PR DESCRIPTION
**Pull request by mcasperson**
# About

This is a post about how to use the NGINX Docker image.

# Release
This post is not release specific.

# Blog image idea

An image has been copied from an existing post.

# Pre-requisites

- [x] The draft is complete and this post is ready to be reviewed.
- [x] I'm familiar with the [writing for Octopus TL;DR](https://www.octopus.design/writing/writing-tldr/).
- [x] The PR build passes validation, and if it doesn't, I've checked the [common validations errors](https://style.octopus.com/writing-at-octopus-tldr#common-validation-errors) and none of those apply.
- [x] The images I've included follow the [rules for working with images](https://www.octopus.design/writing/working-with-images/).
- [ ] I've added alt text for all the images I've included (alt text describes image to people unable to see it, 125 characters max)
- [ ] One screenshot (if any are included) is suitable for Twitter. This screenshot should be 16:9 ratio and at least 1200 x 675 pixels. For screenshots from the Octopus Web Portal, please use Chrome and the browser’s zoom function to remove as much white space as possible. If screenshots are annotated (e.g., with boxes or arrows), please also provide a version that is unannotated for use on Twitter. Do not use dark mode for the Octopus Web Portal.

### CODE REVIEW

File: blog/2022-q3/using-nginx-image/index.md [hunk 1]
```diff
@@ -0,0 +1,267 @@
...
+bannerImageAlt: Man sitting on top of container with green circle with a power up icon
+isFeatured: false
+tags: 
+  - Containers
+  - Docker
+---
+
+Docker is a compelling platform to package and run web applications, especially when paired with one of the many Platform as a Service (PaaS) offerings provided by cloud platforms. NGINX has long provided DevOps teams with the ability to host web applications on Linux, and also provides an official Docker image to use as the base for custom web applications. 
+
+In this post I'll explain how DevOps teams can use the [NGINX Docker image](https://hub.docker.com/_/nginx) to build and run web applications on Docker.
```
Comment (TeganAli) [inline blog/2022-q3/using-nginx-image/index.md:18]:
```suggestion
In this post, I explain how DevOps teams can use the [NGINX Docker image](https://hub.docker.com/_/nginx) to build and run web applications on Docker.
```

**Review by wordlee — APPROVED** [review]
minor line edits throughout.

File: blog/2022-q3/using-nginx-image/index.md [hunk 2]
```diff
@@ -0,0 +1,267 @@
...
+description: Learn how to create Docker web apps based on the NGINX image.
+author: matthew.casperson@octopus.com
+visibility: public
+published: 2022-09-06-1400
+metaImage: blogimage-gettingstartedcontainerisation-2022.png
+bannerImage: blogimage-gettingstartedcontainerisation-2022.png
+bannerImageAlt: Man sitting on top of container with green circle with a power up icon
+isFeatured: false
+tags: 
+  - Containers
```
Comment (TeganAli) [inline blog/2022-q3/using-nginx-image/index.md:12]:
```suggestion
  - DevOps
  - Containers
```

File: blog/2022-q3/using-nginx-image/index.md [hunk 3]
```diff
@@ -0,0 +1,268 @@
...
+## Resources
+
+* [Octopus trial](https://octopus.com/start)
+* [NGINX Docker Image](https://hub.docker.com/_/nginx)
+* [NGINX Docker Image Source Code](https://github.com/nginxinc/docker-nginx)
+* [Dockerfile reference](https://docs.docker.com/engine/reference/builder/)
+
+## Learn more
+
+If you'd like to build and deploy containerized applications to AWS platforms such as EKS and ECS, try the [Octopus Workflow Builder](https://octopusworkflowbuilder.octopus.com/#/). The Builder populates a GitHub repository with a sample application built with GitHub Actions workflows and configures a hosted Octopus instance with sample deployment projects demonstrating best practices such as vulnerability scanning and Infrastructure as Code (IaC).
```
Comment (TeganAli) [inline blog/2022-q3/using-nginx-image/index.md:266]:
```suggestion
If you want to build and deploy containerized applications to AWS platforms such as EKS and ECS, try the [Octopus Workflow Builder](https://octopusworkflowbuilder.octopus.com/#/). The Builder populates a GitHub repository with a sample application built with GitHub Actions workflows and configures a hosted Octopus instance with sample deployment projects demonstrating best practices such as vulnerability scanning and Infrastructure as Code (IaC). 
```

**Pull request merged**